src/utils.test.js

```diff
@@ -876,24 +863,59 @@ describe('utils', () => {
     expect(isEmptyString({})).toEqual(false);
   });
 
-  test('isValidHttpRequestBody - simple flow', () => {
-    expect(isValidHttpRequestBody('BODY')).toEqual(true);
-    expect(isValidHttpRequestBody(Buffer.from('BODY'))).toEqual(true);
+  test('runOneTime -> simple flow', () => {
+    let i = 0;
+    let addToI = () => {
+      i++;
+    };
+    const wrappedAddToI = runOneTimeWrapper(addToI, this);
+    wrappedAddToI();
+    wrappedAddToI();
+
+    expect(i).toEqual(1);
   });
 
-  test('isValidHttpRequestBody -> empty flow', () => {
-    expect(isValidHttpRequestBody()).toEqual(false);
-    expect(isValidHttpRequestBody(0)).toEqual(false);
-    expect(isValidHttpRequestBody([])).toEqual(false);
-    expect(isValidHttpRequestBody({})).toEqual(false);
-    expect(isValidHttpRequestBody(undefined)).toEqual(false);
-    expect(isValidHttpRequestBody(null)).toEqual(false);
+  test('runOneTime -> without context', () => {
```

Comment:
What is the difference between this test and the test above?

Reply:
runOneTimeWrapper(addToI) vs runOneTimeWrapper(addToI,this)